package.json

```diff
@@ -3,13 +3,34 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "npm run clean && webpack -p",
+    "start": "webpack --watch",
+    "server": "npm run clean && webpack && wait-on dist/index.html && browser-sync dist/ --watch",
```

Comment:
OPTIONAL: You could also try [webpack-dev-server](https://github.com/webpack/webpack-dev-server)